pkg/policy/rule.go

```diff
@@ -74,30 +75,78 @@ func (r *rule) validate() error {
 	return nil
 }
 
-func mergeL4Port(ctx *SearchContext, r api.PortRule, p api.PortProtocol, dir string, proto string, resMap L4PolicyMap) int {
-	fmt := p.Port + "/" + proto
-	v, ok := resMap[fmt]
+func adjustL4PolicyIfNeeded(fromEndpoints []api.EndpointSelector, policy *L4Filter) bool {
+
+	if len(policy.FromEndpoints) == 0 && len(fromEndpoints) > 0 {
+		log.Debugf("skipping this L4 policy since it is already covered by an existing policy.")
```

Comment:
"this" is vague for a log message; which L4 Policy are we skipping? The log message should reflect which policy is being skipped.

Reply:
I'll update it to print the policy and the fromEndpoints.